.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Description
+
+
+## Related issues
+
+*
+
+## Related code and/or documentation pull requests
+
+*
```

Comment:
Maybe prefill with "none" or add a comment saying "Remove this section if not needed"

Reply:
done